core/controllers/question.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controller for Question object."""
+
+from core.controllers import base
+from core.domain import question_services
+import feconf
+
+class QuestionsBatchHandler(base.BaseHandler):
+    """This handler completes requests for questions batch."""
+
+    def post(self, collection_id):
```

Comment:
This method needs a test.

Reply:
I have a question regarding this, should this controller be exposed to non-users as well?
As far as I think there is a method can_user_answer_questions, which needs user_id. I am assuming we need user to be logged in to get questions. Correct me if I am wrong.

Reply:
Yep, let's have this be logged-in for now.

Btw, it should be a GET rather than a POST, right?

Reply:
Yea, that's correct. I will update that in the next push.

Reply:
Updated. but I am kinda stuck with the error.

Reply:
Updated. Solved the issue.